drake/systems/plants/RigidBody.h

```diff
@@ -324,6 +320,10 @@ class DRAKERBM_EXPORT RigidBody {
 #endif
 
  private:
+  // TODO(tkoolen): It's very ugly, but parent, dofnum, and pitch also exist
```

Comment:
@tkoolen I placed your name here based on what I observed from git blame. Please let me know if you were not the original author.